Gaijinizer patch/Map220.txt

```diff
@@ -404,109 +446,123 @@ strategies are visible. As I was ...#MTLed
 
 動画投稿サイトを使って、編集動画を投稿している。
 > CONTEXT: Map220/events/1/pages/0/231/Dialogue
-The broadcast on the human realm is. As it is no good if
-you do the radio wave jack. You are posting an edited video
-using a video posting site.#MTLed
+For broadcasting in the Human World, hijacking
+the radio waves is of course not an option. So we
+use a video upload site to post the edited videos.#Strate
 > END STRING
 
 > BEGIN STRING
 正直、逆レイプSHOWの時より規模は縮小しているが
 
  ニーナは、毎日とても楽しそうだ……
 > CONTEXT: Map220/events/1/pages/0/236/Dialogue
-Honestly, the scale is smaller than at the time of Reverse
-rape SHOW. Nina looks very fun every day ...#MTLed
+Honestly, the scale is much smaller than that of
+the Reverse Rape SHOW, but...
+
+Nina seems to be having fun every day......#Strate
 > END STRING
 
 > BEGIN STRING
 よしっ！
 
  今日の分のアップロード終わりっ♪
 > CONTEXT: Map220/events/1/pages/0/243/Dialogue/enemy19
-Good! Today's upload is over♪#MTLed
+Alright!
+
+Today's upload is done♪#Strate
 > END STRING
 
 > BEGIN STRING
 \\N[1]、今日もお疲れ様！
 
  久々のバーゲストちゃんの相手、疲れたでしょ？
 > CONTEXT: Map220/events/1/pages/0/256/Dialogue/enemy19
-\\N[1], thank you very much today!
-Aren't you tired of the long-time Barghest-chan
-partner?#MTLed
+Good work today, \\N[1]!
+
+Are you tired after being Barghest-chan's partner
+for so long?#Strate
 > END STRING
 
 > BEGIN STRING
 つ、疲れたっていうか……動揺しちゃった……
 
 やっぱり魔力のない身体だと、あれが普通なんだね…
 > CONTEXT: Map220/events/1/pages/0/261/Dialogue/kazuya
-I was tired...I got upset ...
-After all if it is a body without magic, that is
-normal ...#MTLed
+A-am I tired......? I'm shaken......
+
+That's normal for a body without magic, after
+all...#Strate
 > END STRING
```

Comment:
Ron Note

Reply:
"N-Not so much tired as shaken..."
"I-I'm more shaken up than tired..."

Reply:
Okay then, so let's do this:
```
I-I'm more shaken up than tired...

That's normal for a body without magic, after
all...
```